vignettes/derive_single_dose_fillin.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "Derive a single dose dataset with imputations"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+ %\VignetteIndexEntry{Derive a single dose dataset with imputations}
+ %\VignetteEngine{knitr::rmarkdown}
+ %\VignetteEncoding{UTF-8}
+---
+  
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+# Introduction
+
+For either single drug administration records,
+or multiple drug administration records covering a range of dates,
+fill-in of missing treatment end datetime (`EXENDTC`) by substitution
+with an acceptable alternate, for example date of death, date of datacut
+may be required. This programming strategy allows the maximum possible number
+of single dose records to be derived.
+
+The example R script requires the date of datacut `DCUTDT` to be specified correctly,
+or if not appropriate to use `DCUTDT` as missing treatment end data
+and missing treatment end datetime could set equal to treatment start date
+and treatment start datetime.
+ADSL variables `DTHDT` and `DCUTDT` are preferred for imputation use.
+
+All available trial treatments are included, allowing multiple different
+last dose variables to be created in ad_adae.R if required.
+
+**Note**: `ADSl` is prerequisite.
+## Required packages.
```

Comment:
Please add an empty line before this one. Otherwise the heading is not rendered correctly.

Reply:
I made the update